src/nft/components/profile/view/ProfileAccountDetails.tsx

```diff
@@ -0,0 +1,40 @@
+import { Trans } from '@lingui/macro'
+import { useWeb3React } from '@web3-react/core'
+import Identicon from 'components/Identicon'
+import { MouseoverTooltip } from 'components/Tooltip'
+import useCopyClipboard from 'hooks/useCopyClipboard'
+import { Box } from 'nft/components/Box'
+import { Row } from 'nft/components/Flex'
+import { caption, headlineLarge, lightGrayOverlayOnHover } from 'nft/css/common.css'
+import { useCallback } from 'react'
+import { Copy } from 'react-feather'
+import { shortenAddress } from 'utils'
+
+export const ProfileAccountDetails = () => {
+  const { account, ENSName } = useWeb3React()
+  const [isCopied, setCopied] = useCopyClipboard()
+  const copy = useCallback(() => {
+    setCopied(account || '')
+  }, [account, setCopied])
+
+  return (
+    <Row className={headlineLarge} marginBottom="48" gap="12">
+      <Identicon size={44} />
+      <Box textOverflow="ellipsis" overflow="hidden">
+        {ENSName || shortenAddress(account ?? '')}
```

Comment:
honestly this component should just return `null` if account is undefined

Reply:
also `??` is better than `||` when comparing nullish values

Reply:
We have a higher level check on sell.tsx to not show this page when account is null, this check was more to resolve compile warnings